### PR TITLE
E1703: Additions to Logging in Expertiza

### PR DESCRIPTION
**Pull request by AswinAK**
Pull request for our final project.

### CODE REVIEW

File: app/helpers/questionnaire_helper.rb [hunk 1]
```diff
@@ -1,99 +1,24 @@
 # OSS808 Change 28/10/2013
 # FasterCSV replaced now by CSV which is present by default in Ruby
 # require 'fastercsv'
-# require 'csv'
+ require 'csv'
```
Comment (yangsong8) [inline app/helpers/questionnaire_helper.rb]: What's the reason that you change this file?

File: app/models/questionnaire.rb [hunk 2]
```diff
@@ -92,4 +93,37 @@ def validate_questionnaire
     results = Questionnaire.where(["id <> ? and name = ? and instructor_id = ?", id, name, instructor_id])
     errors.add(:name, "Questionnaire names must be unique.") if !results.nil? and !results.empty?
   end
+
+  def to_csv(ques)
```
Comment (yangsong8) [inline app/models/questionnaire.rb]: This is not related to the final project, right?

Comment (AswinAK) [inline app/models/questionnaire.rb]: No, this is not related to our project

File: app/helpers/questionnaire_helper.rb (hunk 1 above)
Comment (AswinAK) [inline app/helpers/questionnaire_helper.rb]: This was done by mistake, not part of our changes.
